### DISCUSSION TITLE
Alternative to `MouseDevice?.Position` in master/nightly?

### DISCUSSION THREAD

**Discussion by AsafMahlev** [AvaloniaUI/Avalonia#8713] [Q&A]
I'm trying to translate https://github.com/AvaloniaUI/Avalonia.HtmlRenderer to work on nightly, and the only thing that remains is some lines like these:

```c#
               var pos = (_control.GetVisualRoot() as IInputRoot)?.MouseDevice?.Position ?? default(PixelPoint);
                return Util.Convert(pos);
```
IInputRoot does not longer have MouseDevice, and I couldn't find another api that gives you the position (the new `MouseDevice` does not have this method, and `Pointer` doesn't either.

**Comment by maxkatz6** [comment]
There won't be any. And MouseDevice property was broken by design without position of any other pointer types. 
Subscribe on PointerMoved event and save latest position, as an option.

**Reply by AsafMahlev** [reply]
Thanks! I have updated the branch - https://github.com/AvaloniaUI/Avalonia.HtmlRenderer/pull/17
